src/plugins/controls/jest_setup.ts

```diff
@@ -8,7 +8,7 @@
 
 // Start the services with stubs
 import { pluginServices } from './public/services';
-import { registry } from './public/services/plugin_services.story';
+import { registry } from './public/services/plugin_services.stub';
```

Comment:
The Storybook has been broken for... awhile (not sure exactly when or why they stopped working). However, I need this `jest_setup` to run for the `options_list_popover.test.tsx` tests now that I've added a call to the `optionsList` service in the popover component - and specifically I need the `stub` version of the services because of the changes I made to `data_views.stub.ts`. 

I plan on fixing the Storybooks in a follow-up PR, so I'll need to figure out a better way of handling the two versions at that point. Hopefully for now, this change is okay :+1: